trace-gradle-plugin/src/functionalTest/java/io/bitrise/trace/plugin/util/FunctionalTestHelper.java

```diff
@@ -0,0 +1,298 @@
+     */
+    @NonNull
+    private String getTestNameBorderLogLine(final int lineLength) {
+        final StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("+");
+        stringBuilder.append(concatWord("=", lineLength));
+        stringBuilder.append("+");
+        stringBuilder.append("\n");
+        return stringBuilder.toString();
+    }
+
+    /**
+     * Prints a line with empty content.
+     *
+     * @param lineLength the length of the lines.
+     * @return the line.
+     */
+    @NonNull
+    private String getTestNameEmptyLogLine(final int lineLength) {
+        final StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("|");
+        stringBuilder.append(concatWord(" ", lineLength));
+        stringBuilder.append("|");
+        stringBuilder.append("\n");
+        return stringBuilder.toString();
+    }
+
+    /**
+     * Prints the line with the name of the test.
+     *
+     * @param testName      the given text to display.
+     * @param paddingLength the length of padding for the line, excluding starting and ending cahars.
+     * @return the line.
+     */
+    @NonNull
+    private String getTestNameLogLine(final String testName, final int paddingLength) {
+        final StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("|");
+        stringBuilder.append(concatWord(" ", paddingLength / 2));
+        stringBuilder.append(testName);
+        stringBuilder.append(concatWord(" ", paddingLength / 2));
+        stringBuilder.append("|");
+        stringBuilder.append("\n");
+        return stringBuilder.toString();
+    }
+
+    /**
+     * Concats the given word the given amount of times.
+     *
+     * @param word  the word to concat.
+     * @param times the number of times it should be concatenated.
+     * @return the concatenated String.
+     */
+    @NonNull
+    private String concatWord(@NonNull final String word, final int times) {
+        return IntStream.range(0, times).mapToObj(i -> word).collect(Collectors.joining(""));
+    }
+
+    /**
+     * Gets the name of the current test's directory.
+     *
+     * @param testName the name of the test.
+     * @return the name of the test's directory.
+     */
+    @NonNull
+    public String getTestDirName(@NonNull final TestName testName) {
+        return testProjectDir.getAbsolutePath() + "/" + testName.getMethodName() + "/";
+    }
+
+    /**
+     * Gets the the current test's directory.
+     *
+     * @param testName the name of the test.
+     * @return the test's directory.
+     */
+    @NonNull
+    public File getTestDir(@NonNull final TestName testName) {
+        return new File(getTestDirName(testName));
+    }
+
+    /**
+     * Force-delete an existing folder and its contents, located at {@link #testProjectDir}
+     *
+     * @param testName the name of the test.
+     */
+    public void deleteTemporaryProject(@NonNull final TestName testName) {
```

Comment:
nit: potentially rename this to `deleteTestProjectDir` as it's the opposite of the one where it `setsUpTestDir` - see L189? possible L189 should be `setUpTestProjectDir`?